### ISSUE TITLE
Change Solr to enable a search for dockets for search checkbox option "Exclude pro se petitioner cases"

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#815]

**Comment by mlissner** [comment]
Just converted this from a task on the project board I stumbled upon (weirdly, neither slack nor Github told me that existed).

Anyway, I wanted to make it a task so that I could flag this together with issue #760. That one is very similar to this, and if we're re-importing the entire DB into Solr, we should probably do that one too.

**Comment by jon-freed** [comment]
The project task that prompted this #815 issue's creation is just a placeholder task at this time.  The project, "[Add RECAP search option petitioner represented](https://github.com/freelawproject/courtlistener/projects/4)", is still very much in a design phase, and its active task is centered around PR #808, "Add designs folder and design for proposed petitioner represented attribute".  (That PR is waiting on me to integrate FJC Integrated Database (IDB) information into the design.)  This #815 issue is somewhat duplicative of the issue that preceded the project's creation:  #806, Search: Add options: "Every plaintiff was pro se" or "Plaintiff(s) had representation".

This #815 issue probably isn't hurting anything (AFAIK).  I am noting all of this here to help clarify that it isn't ready for action at this time.

**Comment by mlissner** [comment]
Yeah, figured you were working it all out, but also thought I'd try to slip in another issue if I could. :)

**Comment by mlissner** [comment]
Note that I'm in the process of merging the FJC IDB into the RECAP Archive. In the civil data at least, it has a boolean field for whether a plaintiff, defendant or both are pro se:

```python
    PRO_SE_CHOICES = (
        (PRO_SE_NONE, "No pro se plaintiffs or defendants"),
        (PRO_SE_PLAINTIFFS, "Pro se plaintiffs, but no pro se defendants"),
        (PRO_SE_DEFENDANTS, "Pro se defendants, but no pro se plaintiffs"),
        (PRO_SE_BOTH, "Both pro se plaintiffs & defendants"),
    )
```
